README.md

```diff
@@ -1 +1,146 @@
 # MongoDB for OpenShift - Docker images
+
+This repository contains Dockerfiles for MongoDB images for OpenShift.
+Users can choose between RHEL and CentOS based images.
+
+### Versions
+
+MongoDB versions currently supported are:
+
+* mongodb-2.4
+
+RHEL versions currently supported are:
+
+* RHEL7
+
+CentOS versions currently supported are:
+
+* CentOS7
+
+
+## Installation
+Choose between CentOS7 or RHEL7 based image:
+
+*  **RHEL7 based image**
+
+	This image is not available as automated build in [Docker Index](https://index.docker.io).
+
+	To build a rhel7-based image, you need to run Docker build on a properly subscribed RHEL machine.
+
+	```console
+	git clone https://github.com/openshift/mongodb.git
+	cd mongodb
+	make build TARGET=rhel7
+	```
+
+*  **CentOS7 based image**
+
+	This image is also available on DockerHub. To download it use:
+
+	```console
+	docker pull openshift/mongodb
+	```	
+
+	To build MongoDB image from scratch use: 
+
+	```console
+	git clone https://github.com/openshift/mongodb.git
+	cd mongodb
+	make build
+	```
+
+## Usage
+
+### Environment variables
+
+The image recognizes following environment variables that you can set
+during initialization, by passing `-e VAR=VALUE` to the Docker run
+command.
+
+|    Variable name          |    Description                              |   Default  |
+| :------------------------ | -----------------------------------------   | ---------- |
+|  `MONGODB_USER`           | User name for MONGODB account to be created |
+|  `MONGODB_PASSWORD`       | Password for the user account               |
+|  `MONGODB_DATABASE`       | Database name (optional)                    | production |
+|  `MONGODB_ADMIN_PASSWORD` | Password for the admin user (optional)      |
+
+You can also set following mount points by passing `-v /host:/container`
+flag to docker.
+
+|  Volume mount point    | Description            |
+| :--------------------- | ---------------------- |
+|  `/var/lib/mongodb/`   | MongoDB data directory |
+
+
+### Usage
+
+We will assume that you are using the `openshift/mongodb-24-centos7`
+image. Suppose that you want to set only mandatory required environment
+variables and store the database in the `/home/user/database`
+directory on the host filesystem, you need to execute the following
+command:
+
+```console
+docker run -d -e MONGODB_USER=<user> -e MONGODB_PASSWORD=<password> -e MONGODB_DATABASE=<database> -v /home/user/database:/var/lib/mongodb openshift/mongodb-24-centos7
```

Comment:
if you're only setting the mandatory params (as described on line 78), you don't need to set mongodb_database since you made that one optional.